src/models/CommonModel.ts

```diff
@@ -366,14 +366,15 @@ export class CommonModel extends CommonSchema<CommonModel> {
    * @param originalSchema 
    * @param alreadyIteratedModels
    */
+  // eslint-disable-next-line sonarjs/cognitive-complexity
```

Comment:
You probably should use also `// NOSONAR`, because I still see the problem on sonarcloud

Reply:
I am not sure it is possible for a function like this 🤔

Reply:
Even if you add this comment after function like `functioName(...) { // NOSONAR` ? :)

Reply:
That worked 🍻 Thanks!